iwyu.cc

```diff
@@ -898,21 +898,13 @@ class BaseAstVisitor : public RecursiveASTVisitor<Derived> {
         const_cast<CXXDestructorDecl*>(dtor), parent_type, expr);
   }
 
-  // This is to catch assigning template functions to function pointers.
+  // This is to catch binding of function pointers to templates.
   // For instance, 'MyFunctionPtr p = &TplFn<MyClass*>;': we need to
   // expand TplFn to see if it needs full type info for MyClass.
   bool TraverseDeclRefExpr(clang::DeclRefExpr* expr) {
     if (!Base::TraverseDeclRefExpr(expr))  return false;
     if (CanIgnoreCurrentASTNode())  return true;
 
-    // If it's a normal function call, that was already handled by a
-    // CallExpr somewhere.  We want only assignments.
```

Comment:
I took that from `VisitCallExpr` on line 2334. Should update the commit message to say *Visit*CallExpr.